packages/yourloops/app/error.tsx

```diff
@@ -68,37 +68,58 @@ function OnError(props: OnErrorProps): JSX.Element {
   }, []) // eslint-disable-line react-hooks/exhaustive-deps
 
   const handleOK = (): void => {
-    // Then reload the page
     window.location.replace('/')
   }
 
   const handleShowMore = (): void => {
     setShowMore(true)
   }
 
-  let moreInfos: JSX.Element | null = null
+  let moreInfos: JSX.Element | null
   if (showMore) {
     const error = props.error ? `Error: ${errorMessage}\nStack: ${props.error.stack}` : 'N/A'
     const info = `${(props.event as string).toString()}\nSource: ${props.source}:${props.lineno}:${props.colno}\n${error}`
     moreInfos = (
       <React.Fragment>
-        <DialogContentText style={{ marginTop: '1em', marginBottom: '0px' }} color="textSecondary">{t('app-crash-info')}</DialogContentText>
-        <TextField id="dialog-app-crash-technical-info" style={{ width: '100%' }} inputProps={{ color: 'grey' }} color="secondary" multiline rows={4} value={info}/>
+        <DialogContentText className="no-margin-bottom">
+          {t('app-crash-info')}
+        </DialogContentText>
+        <TextField
+          id="dialog-app-crash-technical-info"
+          inputProps={{ color: 'grey' }}
```

Comment:
Question: i presume it is an inline ? you pass a new object as props

Reply:
No this one is not inline style, it's MUI that proposes this prop to override subcomponent style. TextField is a high level component that embbed several components including an Input. This is the way to override its CSS properties (binded to a CSS class, not inline style)